### PR TITLE
add fff

### PR DESCRIPTION
**Pull request by chris1735**
i'm add a file named 'fff'
good

### CODE REVIEW

**Comment by chris1735** [comment]
i'm waiting for your review

File: fff [hunk 1]
```diff
@@ -0,0 +1 @@
+fff
```
Comment (ipipip1735) [inline fff]: minus a f become two 'f'

Comment (ipipip1735) [inline fff]: agree?

Comment (chris1735) [inline fff]: ok

**Comment by chris1735** [comment]
i have been rewriting

File: fff [hunk 2]
```diff
@@ -0,0 +1,2 @@
+ff
+go go go
```
Comment (ipipip1735) [inline fff]: ok, thanks.

Comment (ipipip1735) [inline fff]: i see

File: fff [hunk 3]
```diff
@@ -0,0 +1,2 @@
+ff
```
Comment (ipipip1735) [inline fff]: yes

Comment (ipipip1735) [inline fff]: you are good

Comment (chris1735) [inline fff]: haha

Comment (chris1735) [inline fff]: yeah

**Pull request merged**